### PR TITLE
POL-156 update azure readme

### PR DESCRIPTION
**Pull request by cdwilhelm**
### Description

* update the Azure policies missing instructions for creds.
* fixing MD for lint compliance 

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: operational/azure/azure_migrate/README.md [hunk 1]
```diff
@@ -25,6 +25,21 @@ This policy has the following input parameters required when launching the polic
 
 - Populate an Azure Migrate project with the discovered servers from a RISC Foundations Assessment
 
+## Prerequisites
```
Comment (Haydenmarx) [inline operational/azure/azure_migrate/README.md]: Should this be combined with the previous 'Pre-requisites' section or should that be renamed?

Comment (cdwilhelm) [reply operational/azure/azure_migrate/README.md]: good catch.  I moved the new items to the existing list.

Comment (Haydenmarx) [reply operational/azure/azure_migrate/README.md]: We could probably also get rid of the dash.

File: compliance/azure/azure_long_stopped_instances/README.md [hunk 2]
```diff
@@ -8,26 +8,41 @@ This policy checks all Azure instances that have been stopped for more than a sp
...
+  - `AZURE_APPLICATION_ID`
+  - `AZURE_APPLICATION_KEY`
+
+## Installation
+
+1. Follow steps to [Create an Azure Active Directory Application](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#create-an-azure-active-directory-application)
+1. Grant the Azure AD Application access to the necessary subscription(s)
+1. [Retrieve the Application ID & Authentication Key](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-application-id-and-authentication-key)
+1. Create RightScale Credentials with values that match the Application ID (Credential name: `AZURE_APPLICATION_ID`) & Authentication Key (Credential name: `AZURE_APPLICATION_KEY`)
+1. [Retrieve your Tenant ID](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-tenant-id)
```
Comment (Haydenmarx) [inline compliance/azure/azure_long_stopped_instances/README.md]: #get-tenant-id seems to be outdated.

File: operational/azure/azure_migrate/README.md (hunk 1 above)
Comment (cdwilhelm) [reply operational/azure/azure_migrate/README.md]: removed the dash.

**Pull request merged**